plugins/foreman_openscap/0.6/index.md

```diff
@@ -148,7 +148,21 @@ Edit ```/etc/foreman-proxy/settings.d/openscap.yml``` with the appropriate setti
 
     puppet module install theforeman-foreman_scap_client
 
-This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and triggering of foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
+This puppet module will *automatically* install foreman_scap_client (if not installed) and configure the client's ```/etc/foreman_scap_client/config.yaml``` with all parameters needed for the operation of foreman_scap_client. The module is used by Foreman to automate the configuration and triggering of foreman_openscap_client, so you should install it in all puppet environments that your hosts use.
+
+Once installed, Foreman will assign this class to hosts or host groups and populate the necessary parameters based on the settings you apply in Foreman under **COMPLIANCE**.
+
+Starting with puppet-foreman_scap_client 0.3.15 shipped with Foreman 1.15 the Foreman plugins yum repo can be set up if you define at least Foreman's major release version.
```

Comment:
I'm sorry, I know I asked you to put these versions in here but the version that contains the change is 0.3.14 and is now planned to be put in 1.14 repositories. I was expecting 1.15 to land earlier... Packaging change is not merged yet but if you could change this, I'll merge the this PR that otherwise looks good.